tests/unit/s2n_client_hello_retry_test.c

```diff
@@ -946,13 +954,6 @@ int main(int argc, char **argv)
             };
             client_conn->security_policy_override = &security_policy_test_tls13_retry_with_pq;
 
-            /* Setup all extensions */
```

Comment:
You're removing the "setup all extensions" comment from some of the code that sets up all extensions. These look like completely random APIs without the context of why they're all being called.

Is there any way you can keep all this code together? Maybe set the config in the middle of this block? Failing that, can you duplicate the comment?